static/app/views/performance/data.tsx

```diff
@@ -380,6 +381,8 @@ export const PERFORMANCE_TERMS: Record<PerformanceTerm, TermFormatter> = {
   mostIssues: () => t('The most instances of an issue for a related transaction.'),
   mostTimeSpentDbQueries: () =>
     t('Database spans on which the application spent most of its total time.'),
+  mostTimeConsumingResources: () =>
+    t('Render blocking resources on which the application spent most of its total time.'),
```

Comment:
We'll have to be careful with displaying only render blocking here, as some projects may not have many. Let's keep it for now tho, as it should be an easy change.

Reply:
Also to add to this, I just thought of this now, but resource module only supports JS/CSS now, we might need to change the titles here. Can address seperately.

Reply:
Yep this can be a quick change for later